ExtraKeyManager.cpp

```diff
@@ -13,6 +13,7 @@
 #include <sstream>
 #include <algorithm>
 #include <cwctype>
+#include <boost/locale/collator.hpp>

```

Comment:
I assume that this boost related change (and the following one as well) is because we upped the boost version?

Reply:
I think that it is for ensuring that string comparison or sorting is achieved correctly and the app behavior is the expected across different regions or cultures.